code-of-conduct/index.md

```diff
@@ -4,32 +4,43 @@ title: Code of Conduct
 description: Our Code of Conduct is at the heart of our event.
 ---
 
-All attendees, speakers, sponsors, and volunteers at {{site.title}} and throughout a mentorship are required to agree with the following code of conduct. Organizers will enforce this code throughout {{site.title}} and for the duration of a mentorship. We are expecting cooperation from all participants to help ensure a safe environment for everybody.
+
+All attendees, speakers, sponsors, and volunteers at Ela Conf, throughout a mentorship, on Slack, and community events are required to agree with the following code of conduct. Organizers will enforce this code throughout Ela Conf, the community, and for the duration of a mentorship. We are expecting cooperation from all participants to help ensure a safe environment for everybody.
 
 Ela is a conference and community for adult (18+) marginalized genders.
 
 ## Need Help?
 
-The {{site.title}} email is [{{site.email}}](mailto:{{site.email}}). Relevant phone numbers will be shared before the conference and during the mentorship matching phase.
+The Ela email is [{{site.email}}](mailto:{{site.email}}). Relevant phone numbers will be shared before the conference and during the mentorship matching phase.
+
+During Ela Conf and community events, you can reach out to the organizers for help:
+
+* [Joni Trythall](/organizers/joni-trythall/)
+* [LeeAnn Kinney](/organizers/leeann-kinney/)
+* [Katy DeCorah](/organizers/katy-decorah/)
+* [Arti Walker-Peddakotla](/organizers/arti-walker-peddakotla/)
+* [Shanise Barona](/organizers/shanise-barona/)
+
+During Ela Conf, at least one organizer will be at the registration table at all times.
```

Comment:
Do we definitely want to do this? It will mean missing some of the main stage talks which is fine, we just hadn't done this in the past.

Reply:
I wonder if it will be sufficient enough to say we will be introduced prior to the event and wearing organizer badges so that people can identify us.

Reply:
I think this is doable! I don't mind volunteering here as when I upload the videos to youtube I'll end up watching them 😛 . I remember keeping an eye on the registration table for at least the morning last year and I'm pretty sure Joni held down the fort for most of the talks the first year.

Reply:
> I wonder if it will be sufficient enough to say we will be introduced prior to the event and wearing organizer badges so that people can identify us.

@shanisebarona I think this is also a good thing to mention! I'll add!